kuksa-client/kuksa_client/__main__.py

```diff
@@ -527,11 +511,35 @@ def connect(self):
             if self.commThread is not None:
                 self.commThread.stop()
                 self.commThread = None
-        config = {'ip': self.serverIP,
-                  'port': self.serverPort,
-                  'insecure': self.insecure,
-                  'protocol': self.serverProtocol
-                  }
+
+        # Check we have a valid server URI
+        srv = urlparse(self.server)
+        config = {"port": 55555, "insecure": True}
```

Comment:
See comment above, do we want different default for port depending on protocol, i.e. use 8090 if protocol is ws?